src/base/preferences.h

```diff
@@ -122,6 +123,8 @@ class Preferences : public QObject
     void setPreventFromSuspendWhenDownloading(bool b);
     bool preventFromSuspendWhenSeeding() const;
     void setPreventFromSuspendWhenSeeding(bool b);
+    void setTheme(UIThemeManager::Theme theme);
+    UIThemeManager::Theme theme();
```

Comment:
I would place these functions just below `void setLocale(const QString &locale);`, similar to the layout of the option dialog.
And also in .cpp file too.